{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -0,0 +1,224 @@
+    problem_type = recipe_config.get("recipe").split("/")[0]
+    if problem_type.lower() == "regression":
+        return "regressor"
+    elif problem_type.lower() == "classification":
+        return "classifier"
+    else:
+        raise Exception(f"Unsupported recipe {recipe_config}")
+
+def get_targets_from_recipe():
+    recipe_config = get_recipe_config("../", f"databricks-{env}")
+    return recipe_config.get("target_col")
+
+# set model evaluation parameters that can be inferred from the job
+model_uri = dbutils.jobs.taskValues.get("Train", "model_uri", debugValue="")
+model_name = dbutils.jobs.taskValues.get("Train", "model_name", debugValue="")
+model_version = dbutils.jobs.taskValues.get("Train", "model_version", debugValue="")
+
+baseline_model_uri = "models:/" + model_name + "/Production"
+evaluators = "default"
+assert env != "None", "env notebook parameter must be specified"
+assert model_uri != "", "model_uri notebook parameter must be specified"
+assert model_name != "", "model_name notebook parameter must be specified"
+assert model_version != "", "model_version notebook parameter must be specified"
+
+# set experiment
+with open("../databricks-config/output/prod.json", "r") as f:
+    experiment_name = json.load(f)["{{cookiecutter.project_name}}_experiment_name"]["value"]
+mlflow.set_experiment(experiment_name)
+
+# COMMAND ----------
+
+##################################################################################
+# TODO : Please fill in this cell with proper values for enable_baseline_comparison,
+#        data, targets, model_type, custom_metrics, validation_thresholds
+#        evaluator_config
+##################################################################################
+
+# Whether to load the current registered "Production" stage model as baseline. A version with "Production" stage must 
+# exist for the model.
+# Baseline model is a requirement for relative change and absolute change validation rules.
+# TODO(required) : enable_baseline_comparison
+enable_baseline_comparison = False
+
+# model validation data input for staging or prod workspace. A Pandas DataFrame or Spark DataFrame, containing evaluation features and labels.
+# Please refer to data parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+if env == "prod":
+    # TODO(required) : set data for prod workspace
+    data = None
+elif env == "staging":
+    # TODO(required) : set data for staging workspace
+    data = None
+else:
+    raise Exception("Unknown environment. Please select 'prod' or 'staging' as environment name")
+
+# The string name of a column from data that contains evaluation labels.
+# Call get_targets_from_recipe() to get targets from recipe configs if mlflow recipe is used. 
+# Please refer to targets parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(required) : targets
```

Comment:
ditto. This should be `optional`.

Reply:
This line actually needs user to confirm the `targets` value.
I do not want to mark it as optional in case user may think they can skip setting the variable or reading the comment.
What's your recommendation?